internal/generator/testdata/pet-api-doc.json

```diff
@@ -36,6 +36,26 @@
         },
         "type": "array"
       },
+      "google.type.Money": {
```

Comment:
@lsmoura  Do we have to pass the nano to represent the cents in the unit now?

Reply:
According to the proto, yes:

```
message Money {
  // The 3-letter currency code defined in ISO 4217.
  string currency_code = 1;

  // The whole units of the amount.
  // For example if `currencyCode` is `"USD"`, then 1 unit is one US dollar.
  int64 units = 2;

  // Number of nano (10^-9) units of the amount.
  // The value must be between -999,999,999 and +999,999,999 inclusive.
  // If `units` is positive, `nanos` must be positive or zero.
  // If `units` is zero, `nanos` can be positive, zero, or negative.
  // If `units` is negative, `nanos` must be negative or zero.
  // For example $-1.75 is represented as `units`=-1 and `nanos`=-750,000,000.
  int32 nanos = 3;
}
```

Source: https://github.com/googleapis/api-common-protos/blob/main/google/type/money.proto